lib/logstash/outputs/elasticsearch.rb

```diff
@@ -266,6 +266,12 @@ def initialize(*params)
   end
 
   def register
+    if !failure_type_logging_whitelist.empty?
+      @deprecation_logger.deprecated "'failure_type_logging_whitelist' is deprecated and in a future version of " +
```

Comment:
deprecation msg only shows in logstash-deprecation.log which might be overlooked. Can we have a normal log message as well?

Reply:
@andsel ☝️

Reply:
Oh thanks, I missed it

Reply:
I think we could have both, one line in deprecation logger plus a warning line in Logstash log.
The deprecation log should be checked by users (or in some automated way) to see if they are slowly slippering.

Reply:
Fixed by f5397d9048e0d628ad8b1a385a772bcd970e6c68